examples/step-72/doc/intro.dox

```diff
@@ -6,4 +6,68 @@
 
 <h1>Introduction</h1>
 
-<h3>step-15 using automatic differentiation</h3>
+<h3>Motivation</h3>
```

Comment:
Thanks a lot for adding all of this. I really appreciate the time that you spent on this. You've written most of what I would have _wanted_ to add as an introduction to the problem itself, but only much better. May I please ask that you add yourself as an author to the tutorial? The extensions were really trivial, so most of the time is spent documenting the problem.